diff --git a/bin/hubot b/bin/hubot
index f58bef6..adeffa1 100755
--- a/bin/hubot
+++ b/bin/hubot
@@ -2,7 +2,8 @@
 
 set -e
 
+umask 0
 npm install
-export PATH="node_modules/.bin:node_modules/hubot/node_modules/.bin:$PATH"
+export PATH="$PATh:node_modules/.bin:node_modules/hubot/node_modules/.bin:$PATH"
 
 exec node_modules/.bin/hubot --name "issuebot" "$@"
diff --git a/config/slack-github-issues.json b/config/slack-github-issues.json
index a98aa0a..dca7b40 100644
--- a/config/slack-github-issues.json
+++ b/config/slack-github-issues.json
@@ -1,7 +1,7 @@
 {
   "githubUser": "infotroph",
-  "githubApiToken": "<github-api-token>",
-  "slackApiToken": "<slack-api-token>",
+  "githubApiToken": "secretapitoken",
+  "slackApiToken": "secretapitoken",
   "githubTimeout": 5000,
   "slackTimeout": 5000,
   "successReaction": "heavy_check_mark",
diff --git a/entrypoint.sh b/entrypoint.sh
index 33542ac..7af5ff5 100755
--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -2,8 +2,9 @@
 
 set -e
 
+umask 0
 #npm install
-export PATH="node_modules/.bin:node_modules/hubot/node_modules/.bin:$PATH"
+export PATH="$PATh:node_modules/.bin:node_modules/hubot/node_modules/.bin:$PATH"
 
 if [ "$1" == "bot" ]; then
   shift
